cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/RunCommand.java

```diff
@@ -113,6 +120,12 @@ public void execute() {
         if (!argList.isEmpty()) {
             if (!argList.get(0).equals("--")) { // project path provided
                 this.projectPath = Paths.get(argList.get(0));
+                if (RunCommand.JAR_EXTENSION_MATCHER.matches(this.projectPath)) {
+                    CommandUtil.printError(this.errStream, "options are not allowed when executing jar files",
```

Comment:
Since `--debug` is supported, shall we rephrase this in a different way?
Suggestion:
```suggestion
                    CommandUtil.printError(this.errStream, "unsupported option(s) provided for jar execution",
```

Reply:
Changed error to suggested message